CODEOWNERS

```diff
@@ -55,14 +68,208 @@
 /cmake/sysbuild/suit_utilities.cmake      @nrfconnect/ncs-charon
 
 # All doc related files
-/doc/					  @nrfconnect/ncs-co-doc
+/doc/                                     @nrfconnect/ncs-co-doc
```

Comment:
Please remove this one, we need to ensure that we catch missing entries and this will mask them